pytest/lib/branches.py

```diff
@@ -120,6 +121,6 @@ def prepare_ab_test(other_branch):
         if not os.getenv('NAYDUCK'):
             compile_binary(str(other_branch))
         else:
-            print('RC binary should be downloaded for NayDuck.')
+            logger.info('RC binary should be downloaded for NayDuck.')
```

Comment:
`logging.critical`. After all, we’re aborting the script.

Here and throughout the commit where we log before aborting with a `sys.exit(1)`.